examples/k8s.yaml

```diff
@@ -71,6 +71,23 @@ provision:
     rm -f /etc/cni/net.d/*.conf*
     apt-get install -y kubelet kubeadm kubectl && apt-mark hold kubelet kubeadm kubectl
     systemctl enable --now kubelet
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    grep SystemdCgroup /etc/containerd/config.toml && exit 0
+    # Configuring a cgroup driver
+    cat <<EOF >>/etc/containerd/config.toml
```

Comment:
Maybe we also have to ensure that the config.toml has `version = 2` header?

Reply:
Sure, I don't know what to do if it is not - but we could have the script "exit 1"

It has only been tested with the current config:

```toml
  version = 2
  [proxy_plugins]
    [proxy_plugins."stargz"]
      type = "snapshot"
      address = "/run/containerd-stargz-grpc/containerd-stargz-grpc.sock"
```

Not sure why the extra indent ?